scripts/pi-hole/js/taillog.js

```diff
@@ -71,6 +90,9 @@ function getData() {
       }
 
       data.log.forEach(function (line) {
+        // Format line if this is the dnsmasq log
+        if (GETDict.file === "dnsmasq") line.message = formatLine(line.message);
+
         // Add new line to output
         $("#output").append(
           '<span><span style="border-left: 2px red" class="left-line">&nbsp;</span><span class="text-muted">' +
```

Comment:
I noticed you used a `span` to add a margin (using `&nbsp;` space) and tried to add a border (but no border is visible):
```html
<span style="border-left: 2px red" class="left-line">&nbsp;</span>
```

- The border is missing because there is an issue on the CSS style: `border-style` is missing. 
  The CSS should be `2px solid red`, but I'm not sure if each line should have a border. It looks strange:
  ![image](https://github.com/pi-hole/web/assets/1385443/582d7bc7-ef73-48cd-a58a-7daf9a540d01)

  
- Also, you don't need to add spacing/margin using an extra HTML tag. 
  You can achieve the same effect using a simpler markup and adding the style to `pi-hole.css`.


My suggestion - do not use a border and use the following code:

```suggestion
          '<span class="log-entry"><span class="text-muted">' +
```

New CSS:
```
.log-entry {
  padding-left: .5em;
}
```

Reply:
Thanks for your suggestion. The red border on the left seems to be a remnant from the initial implementation where newly added lines were highlighted like this. This was then replaced by adding the fading-out horizontal red line later on. Removing this code was likely forgotten because it didn't show up. I added exactly your recommendation and did this in a separate commit as editing `pihole.css` is not possible via a web change suggestion when the file is untouched.